requirements.txt

```diff
@@ -6,6 +7,7 @@ pytest-mock==1.10.4
 pytest-cov==2.7.1
 pyyaml==5.1.1
 requests==2.22.0
+sqlalchemy==1.3.3
```

Comment:
Pinning sqlalchemy 1.3.3 until https://github.com/kvesteri/sqlalchemy-utils/issues/374 is resolved in SQLAlchemy-Utils